### PR TITLE
Rec index

### PR DESCRIPTION
**Pull request by dpodvyaznikov**
Add `RecIndex` to iterate over receiver gathers.

### CODE REVIEW

File: seismicpro/src/seismic_index.py [hunk 1]
```diff
@@ -434,3 +434,34 @@ def show_heatmap(self, **kwargs):
             show_2d_heatmap(self._idf, **kwargs)
         else:
             show_1d_heatmap(self._idf, **kwargs)
+
+
+class RecIndex(TraceIndex):
```
Comment (a-arefina) [inline seismicpro/src/seismic_index.py]: Let's finally extract SeismicIndex class

**Comment by nikita-klsh** [comment]
Add `RecIndex` to `__init__.py`

File: seismicpro/src/seismic_index.py (hunk 1 above)
Comment (a-arefina) [reply seismicpro/src/seismic_index.py:439]: OK, I will do it myself in my next PR into SeismicPro

**Comment by dpodvyaznikov** [comment]
Unnecessary right now due to planned changes in index.

**Pull request closed without merging**